design-library/src/components/BccTabs/BccTabs.stories.ts

```diff
@@ -22,6 +22,11 @@ export default {
       options: ["sm", "base", "lg"],
       control: { type: "radio" },
     },
+    //Boolean fill
+    fill: {
```

Comment:
Please add this to the `Example.args` below so it properly shows up with default. Now it looks like this instead of a nice False - True slider:

<img width="931" alt="image" src="https://github.com/bcc-code/bcc-design/assets/39056299/956340c2-8a93-41f1-b6b7-03c9ee59ceb2">